### DISCUSSION TITLE
antd/es 和 antd 效果不同

### DISCUSSION THREAD

**Discussion by heyden-suimu** [ant-design/ant-design#43209] [Q&A] [answered]
在项目中，浏览器端
import { ConfigProvider as ConfigProvider$1 } from 'antd';
import { ConfigProvider as ConfigProvider$1 } from 'antd/es';
这两种写法，前者prefix生效，后者prefix不生效，，，

**Comment by afc163** [comment, marked as answer]
不要用后者。你可能引入了两个 ConfigProvider 实例。

**Reply by heyden-suimu** [reply]
是的，确实有两个ConfigProvider实例，但是理论上 我看了入口路径import antd 和 import antd/es貌似走的是一样的，import antd理论上也是走的es在谷歌常规浏览器上，但是咋会出现，后者不生效，，，没搞懂，，
